core/src/builder.spec.ts

```diff
@@ -287,71 +277,6 @@ describe('building', () => {
       );
     });
 
-    it('should build all build upon properties with callbacks', () => {
```

Comment:
We had a feature that allowed properties updaters to provide a value based on the previous one. 
Keeping this feature with the new proposal is quite hard so, since I haven't found any model in this repo using this feature, I'd like to remove it by now.

If we find any MC applications tests actually using this feature, then I'll get back to it if necessary but I would like to forget about it by now if possible.